service/handler.go

```diff
@@ -41,9 +41,15 @@ func ReverseProxyOrResponse(
 	env config.EnvironmentVariables,
 	w http.ResponseWriter,
 	req *http.Request,
-	permission *openapi.RondConfig,
-	partialResultsEvaluators core.PartialResultsEvaluators,
+	evaluatorSdk core.SDKEvaluator,
 ) {
+	var permission openapi.RondConfig
+	var partialResultsEvaluators core.PartialResultsEvaluators
+	if evaluatorSdk != nil {
```

Comment:
how can it be `nil`? I'd panic if nil arrives here, no?

Reply:
The `alwaysProxyHandler` function calls `ReverseProxyOrResponse` with nil. So, it is required at the moment this check to pass tests.